components/brave_wallet_ui/components/extension/sign-panel/index.tsx

```diff
@@ -137,6 +137,9 @@ export const SignPanel = (props: Props) => {
     }
   }, [showWarning])
 
+  // TODO(apaymyshev): implement ui for this.
+  console.log(selectedQueueData.domain)
+
```

Comment:
Discussed with @onyb to have ui here similar to 
![image](https://user-images.githubusercontent.com/168356/215728204-13e55a69-ea9e-488e-a4de-10116c95111c.png)